searx/engines/brave.py

```diff
@@ -291,15 +291,22 @@ def _parse_search(resp):
         if url is None or title_tag is None or not urlparse(url).netloc:  # partial url likely means it's an ad
             continue
 
-        content_tag = eval_xpath_getindex(result, './/div[contains(@class, "snippet-description")]', 0, default='')
+        content = (
+            extract_text(eval_xpath_getindex(result, './/div[contains(@class, "snippet-description")]', 0, default=''))
+            or ''
```

Comment:
`extract_text()` always returns a string when its argument is an `ElementBase` (or a string in the default case), so this `or ''` is redundant. Reference: https://github.com/searxng/searxng/blob/master/searx/utils.py#L208

Reply:
Yeah is redundant .. it was because of the type checker reports a type error .. fixed it ([diff](https://github.com/searxng/searxng/compare/190bfa390c45ef31008cef688375378772be2dde..76e40474c192edddb7a731bee728d44d2b5e1815)) by adding a type to `content: str` and `#type: ignore`.